Mark toolkits that require custom auth and Callout auth providers without a default app (#113)
ArcadeAI/docs: pages/toolkits/entertainment/spotify.mdx

  authType="oauth2"
  authProviderName="Spotify"
  versions={["0.1.0"]}

>   note="This tool is not available when using the arcade-spotify Auth Provider in the Arcade Cloud. To use this tool, you must create your own Auth Provider."

Reviewer: ```suggestion
  note="This toolkit is not available in Arcade Cloud. You can use these tools with a self-hosted instance of Arcade."
```
